generators/generator-base.js

```diff
@@ -1493,7 +1504,12 @@ module.exports = class extends PrivateBase {
      * @param {string} msg - message to print
      */
     info(msg) {
-        this.log.info(msg);
+        const debug = this.getDebug();
+        if (debug.enabled) {
+            debug(`${chalk.green('INFO!')} ${msg}`);
```

Comment:
Logging info message into a debug method wrapper is the real behaviour we want ? I don't know the debug package but what should we do that ?

Reply:
- when stdin is muted like `npm test`.
- to have blueprint namespace printed with the msg.